coderd/database/queries/workspaces.sql

```diff
@@ -267,6 +267,7 @@ WHERE
 	-- Authorize Filter clause will be injected below in GetAuthorizedWorkspaces
 	-- @authorize_filter
 ORDER BY
+	CASE WHEN workspaces.owner_id = @order_by_favorite AND workspaces.favorite THEN 0 ELSE 1 END ASC,
```

Comment:
review: favorites should only show up first for the owner

Reply:
maybe drop a comment in the SQL